JitsiParticipant.js

```diff
@@ -294,6 +308,32 @@ export default class JitsiParticipant {
         this._role = newRole;
     }
 
+    /**
+     * Sets source info.
+     * @param {MediaType} mediaType The media type, 'audio' or 'video'.
+     * @param {boolean} muted The new muted state.
+     * @param {string} sourceName The name of the source.
+     * @param {string} videoType The video type of the source.
+     * @returns {void}
+     */
+    setSources(mediaType, muted, sourceName, videoType) {
```

Comment:
Should we rename it to `_setSources` in order to indicate that this method is not part of the public API of LJM? WDYT?

Reply:
Done